app/assets/stylesheets/components/buttons.scss

```diff
@@ -79,27 +80,28 @@ $outlined-border-width: 2px;
     }
   }
 
-  &--text {
+  &--ghost {
```

Comment:
I'm adding/renaming a button type. It won't have any decorations, so no background, etc..